core/src/main/java/org/wildfly/extras/creaper/core/online/SslOptions.java

```diff
@@ -179,6 +179,7 @@ public Builder trustStoreType(KeyStoreType trustStoreType) {
 
         /**
          * Errors during hostname verification will be ignored.
+         * This is valid only for {@link ManagementProtocol#HTTPS}, otherwise has no effect.
```

Comment:
This is not precise. "Otherwise has no effect" can imply that it's impossible to turn hostname verification _off_ otherwise, except that it's the exact opposite. Maybe "Hostname verification is an HTTPS concept, so this method only makes sense when using {@link ManagementProtocol#HTTPS}. There's no such thing as hostname verification with the native management protocols."